humanizer/operation.py

```diff
@@ -132,6 +154,10 @@ def add_event(self, rest):
             # This is just a catch all until we exhaust the supported verbs
             raise Exception('Unsupported VERB in: {}'.format(rest))
 
+        # Sort the requests details, to make it more deterministic
+        for request in self.requests:
+            request["details"] = sorted(request["details"])
```

Comment:
The lack of a sort on the request details was causing unit-tests to be non-deterministic, this fixes that for unit-testing joy, though it would have still worked because we sorted on the lists in the dict method, which is what the run.py uses.